styles.css

```diff
@@ -501,3 +501,82 @@ section .portfolio {
 .about-container {
   display: flex;
 }
+
+/* Contact me section */
+
+#contact-section {
+  background: #6070ff;
+  padding-bottom: 7rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-bottom-left-radius: 4.6875rem;
+  border-top-right-radius: 7.1875rem;
+  background-image: url(/assets/header-bg.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  /* background-position: 20% 40%; */
+}
+
+#contact-section > h2 {
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  display: flex;
+  align-items: center;
+}
+
+#contact-form {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.font` class in multiple places or simply put the `font-family` property in the `body` selector and all other children elements going to inherit it by default.

- (your code)

```css
#contact-section > h2 {
  /* ...code */
  display: flex;
  /* ...code */
}

#contact-form {
  /* ...code */
  display: flex;
  /* ...code */
}
```

- suggestion

```css
.flex {
  display: flex;
}
```

Reply:
Ok, @MrRamoun I will make sure I apply this going forward, thank you for the help!